tests/setup/kfctl_openshift.yaml

```diff
@@ -14,10 +14,20 @@ spec:
       parameters:
         - name: s3_endpoint_url
           value: "s3.odh.com"
+        - name: notebook_destination
+          value: rhods-notebooks
```

Comment:
@crobby This parameter is set in [odh-deployer(https://github.com/red-hat-data-services/odh-deployer/blob/main/opendatahub.yaml#L16) but it breaks when you run the jupyterhub CI tests manually.  Does openshift-ci use this kfdef for the CI runs or is there a RHODS specific tests kfdef?

Reply:
I need to double check, but I think it's using this kfdef

Reply:
Local test runs work without it so I think it will probably be required for RHODS CI runs in openshift-ci